modules/kueue-release-notes-1.2.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * ai_workloads/kueue/release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="release-notes-1.2_{context}"]
+= Release notes for {kueue-name} version 1.2
+
+[role="_abstract"]
+{kueue-name} version 1.2 is a generally available release that is supported on {product-title} versions 4.18 and later on the 64-bit x86 architecture. {kueue-name} version 1.2 uses link:https://kueue.sigs.k8s.io/docs/overview/[Kueue] version 0.13.
+
+[id="release-notes-1.2-new-features_{context}"]
+== New features and enhancements
+
+Monitoring of pending workloads::
+{kueue-name} version 1.2 provides the `VisibilityOnDemand` feature to monitor the pipeline of pending jobs in the cluster queue and batch users for the local queue, and help users to estimate when their jobs will start. For more information, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.21/html/ai_workloads/red-hat-build-of-kueue#monitoring-pending-workloads.html[Monitoring pending workloads].
```

Comment:
```suggestion
{kueue-name} version 1.2 provides the `VisibilityOnDemand` feature to monitor the pipeline of pending jobs in the cluster queue and the local queue, and help users to estimate when their jobs will start. For more information, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.21/html/ai_workloads/red-hat-build-of-kueue#monitoring-pending-workloads.html[Monitoring pending workloads].
```

Reply:
no need to detail batch users as we don't detail which user is allowed to access the cluster queue.